### PR TITLE
fix: show val if previously set in fileInput

### PR DESCRIPTION
**Pull request by connoratrug**

### CODE REVIEW

File: apps/molgenis-components/src/components/forms/InputFile.vue [hunk 1]
```diff
@@ -47,6 +47,11 @@
         </button>
       </div>
     </div>
+    <div v-if="value">
+      <a v-if="value && value.url" :href="value.url">
+        Previous value: {{ value.name }}.{{ value.extension }}
+      </a>
+    </div>
```
[the remark this answers is not in the document]
Comment (mswertz) [reply apps/molgenis-components/src/components/forms/InputFile.vue:54]: Indeed. in the explorer it simply gets the name of the input (there is no 'name' for a file). 
Should we do the same here?
See https://emx2.dev.molgenis.org/pet%20store/tables/#/User

[the remark this answers is not in the document]
Comment (mswertz) [reply apps/molgenis-components/src/components/forms/InputFile.vue:54]: Might also include size then.

**Review by mswertz — CHANGES_REQUESTED** [review]
Well spotted that this should be possible. I agree with comment of max it now looks odd with only '.csv' so suggest to change by adding simply the name of the input (there is no name of the file).

**Comment by connoratrug** [comment]
This pr is about moving the file component form the styleguide to the molgenis components lib, the "Previous value:.. 
 was missing. The name of the file is part of the upload and should be stored in the backend and part of the response when fetching the file data. suggest to do that fix in a separate PR

File: apps/molgenis-components/src/components/forms/InputFile.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (chinook25) [reply apps/molgenis-components/src/components/forms/InputFile.vue:54]: If there never is a file name here, maybe instead change the message to something like: 'Extension of previous upload: {{value.extension}}'?

**Pull request merged**